requirements.txt

```diff
@@ -27,7 +30,7 @@ jsonschema==2.5.1  # MIT
 lxml==3.6.0  # BSD
 nameparser==0.4.0  # LGPL
 newrelic==2.68.0.50  # newrelic APM agent, Custom License
-pendulum==0.6.2  # MIT
+pendulum==1.2.0  # MIT
```

Comment:
Turns out I had some unfinished `gov.pubmedcentral.pmc` HarvestLogs sitting around, which the new harvest task helpfully picked up! This pendulum upgrade breaks that transformer:
```
  File "/Users/abram/code/SHARE/share/transformers/gov_pubmedcentral_pmc.py", line 331, in get_date_published
    return str(pendulum.from_date(int(year.string), int(month.string), int(day.string)))
AttributeError: module 'pendulum' has no attribute 'from_date'
```